hvpy/io.py

```diff
@@ -54,4 +55,17 @@ def url(self) -> str:
         """
         Final API endpoint URL.
         """
-        return BASE_URL + self.__class__.__name__[:-15] + "/"
+        return base_url + self.__class__.__name__[:-15] + "/"
+
+
+def set_api(url: Optional[str] = None) -> None:
+    """
+    Sets the base URL for the Helioviewer python API.
+    """
+    global base_url
```

Comment:
We need a different way that isn't a global variable.

Reply:
Right.